### PR TITLE
fix typo in events.mdx

### PR DESCRIPTION
**Pull request by seeekr**
not sure if I'm making the correct suggestion, but the original code cannot possibly work because there is no `member` variable.

### CODE REVIEW

**Comment by seeekr** [comment]
oh shoot, it should probably be `event.target.data.emailVerified` instead?

**Comment by davenewza** [comment]
> oh shoot, it should probably be `event.target.data.emailVerified` instead?

@seeekr well spotted, and yes, you're correct that it's `event.target.data.emailVerified`

File: pages/events.mdx [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default VerifyEmail(async (ctx, event) => {
 
     case "member.updated":
       // Verify their email address if it was changed.
-      if (!member.emailVerified) {
+      if (!event.data.emailVerified) {
```
Comment (davenewza) [inline pages/events.mdx:119]:
```suggestion
      if (!event.target.data.emailVerified) {
```

**Review by davenewza — CHANGES_REQUESTED** [review]
Please do pop a fix in as per your suggestion

**Review by davenewza — APPROVED** [review]

**Pull request merged**